measureCpu.py

```diff
@@ -10,28 +10,31 @@
 import jc
 
 class MeasureCPU(Task):
-    def __init__(self, tc: TestConfig, node_name: str, tenant: bool):
+    def __init__(self, tc: TestConfig, node_name: str, tenant: bool, is_client: bool = False):
```

Comment:
Let's start adding some commentary on arguments.  I am not sure if the measure CPU plugin needs to be aware that it is measuring client vs. server. Or we can couple it by providing a reference to the Iperf/Traffic Flow instance it wants to measure. Wdyt?

Reply:
Yeah this would not be needed by updating the  TftAggregateOuput to handle plugins abstractly.

This was just to ensure a unique field for the output of each of these plugins: i.e. https://github.com/wizhaoredhat/ocp-traffic-flow-tests/pull/10/files#diff-694b083fdb1c62ebbbbe1e0c047b094b3eb64e1b301494c9d3d0f429c87d4d10R73